### PR TITLE
Initial upload

### PR DESCRIPTION
**Pull request by oliverfoster**
Used to display individual assessment scores.

example: http://rawgit.com/oliverfoster/builds/assessment/index.html

### CODE REVIEW

**Comment by oliverfoster** [comment]
To be made in conjuction with PRs:

assesmentResultsTotal https://github.com/adaptlearning/adapt-contrib-assessmentResultsTotal/pull/1
adapt_framework https://github.com/adaptlearning/adapt_framework/pull/564
assesment https://github.com/adaptlearning/adapt-contrib-assessment/pull/32

File: less/assessmentResults.less [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.article.article-noheader .article-inner {
+	padding-top: initial;
```
Comment (oliverfoster) [inline less/assessmentResults.less]: initial is unsupported in ie8, need to fetch proper default values

**Comment by hrajotia** [comment]
+1

**Comment by zenduo** [comment]
+1

File: properties.schema [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                  "inputType": { "type": "Boolean", "options": [false, true]},
+                  "validators": []
+                },
+                "_scoreToPass": {
+                  "type":"number",
+                  "required":false,
+                  "title": "Pass mark",
+                  "inputType": "Number",
+                  "default": 60,
+                  "validators": ["number", "required"]
```
Comment (dancgray) [inline properties.schema]: Does this contradict the "required":false, attribute?

Comment (oliverfoster) [inline properties.schema]: I haven't done the schemas yet. I don't understand them well enough. I have asked Tom Taylor about it and he generously offered to do them (when he has time).

**Comment by dancgray** [comment]
+1

**Pull request merged**